webgrid_blazeweb_ta/views.py

```diff
@@ -21,5 +21,7 @@ def default(self):
         pg.apply_qs_args()
         if pg.export_to == 'xls':
             pg.xls.as_response()
+        elif pg.export_to == 'xlsx':
+            pg.xlsx.as_response()
```

Comment:
Let's add a `get_renderer` method to the grid, that returns the renderer when given the corresponding attribute name. And, add an `export_as_response` that uses `export_to` to get the renderer and call `as_response`. Here, this becomes:
```
pg.apply_qs_args()
if pg.export_to:
    pg.export_as_response()
```

Which removes much of the manual usage going on.